otter/integration/lib/mimic.py

```diff
@@ -41,25 +43,32 @@ def change_server_statuses(self, rcs, ids_to_status):
             pool=self.pool
         ).addCallback(check_success, [201]).addCallback(self.treq.content)
 
+    @inlineCallbacks
     def sequenced_behaviors(self, rcs, criteria, behaviors,
-                            event_description="creation"):
+                            event_description="creation",
+                            add_cleanup_to=None):
```

Comment:
Non-blocking thought: Do you think it would help if `MimicNova` constructs with test case object? This way it can always cleanup with that object.